deploy/operator.yaml

```diff
@@ -47,3 +47,6 @@ spec:
                   fieldPath: metadata.namespace
             - name: OPERATOR_NAME
               value: "marketplace-operator"
+            - name: "RELEASE_VERSION"
+              # The string "0.0.1-snapshot" is substituted by the CVO with the version of the payload
```

Comment:
This won't happen given this yaml is used only by the SDK. I suggest removing the comment and leaving the rest for consistency.

I am also not sure if we need this here. @kevinrizza what do you think?

Reply:
@aravindhp I think it doesn't hurt to keep these yamls in sync. If we ever updated our build process somehow to actually use the sdk to build, then it would be useful not to have to go back and make all these changes. But I am also okay with not doing it.

Reply:
Let's keep it in sync then